jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -39,17 +39,14 @@
 /** Immutable configuration options for the container. */
 public class ContainerConfiguration {
 
-  /** The default creation time of the container (constant to ensure reproducibility by default). */
-  public static final Instant DEFAULT_CREATION_TIME = Instant.EPOCH;
```

Comment:
I wonder if we should keep this?

Reply:
Okay. The class is not a public API, so I moved it to the builder and made it `private`.

FYI, eventually we should remove the entire class and replace it with `ContainerBuildPlan`. They are basically identical.